packages/extension/src/companion/App.tsx

```diff
@@ -32,10 +33,15 @@ export default function App({
   const [isOptOutCompanion, setIsOptOutCompanion] = useState<boolean>(
     settings?.optOutCompanion,
   );
+
   if (isOptOutCompanion) {
     return <></>;
   }
 
+  const memoizedFlags = useMemo(() => {
+    return flags;
+  }, [flags]);
```

Comment:
Just my favorite, not required.

```suggestion
  const memoizedFlags = useMemo(() => flags, [flags]);
```